modules/metrics/src/main/java/org/apache/ignite/internal/metrics/MetricRegistry.java

```diff
@@ -274,7 +275,7 @@ private void removeMetricSet(String srcName) {
      *
      * @param metricSets New map of metric sets that should be saved to new version of metric snapshot.
      */
-    private void updateMetricSnapshot(Map<String, MetricSet> metricSets) {
+    private void updateMetricSnapshot(SortedMap<String, MetricSet> metricSets) {
```

Comment:
What is the goal of this change? Could you please clarify this?

Reply:
It was not obvious that there is sorted map which finally is used for exporting the metrics in lexicographical order. I made it more clear.